src/scss/core/_vars.scss

```diff
@@ -7,3 +7,35 @@ $grid-breakpoints: (
   lg: 992px,
   xl: 1200px
 );
+
+$grid-gutter-width-lg: 30px;
+
+//colors
+$primary: #f3436c;
+$secondary: #00635b;
+$body-color: #151515;
+$light-pink: #f5dadf;
+$light-green: #e0f0e9;
+$gray: #4e4e4e;
+
+//font size
+$h1-font-size: 1.5em;
+$h2-font-size: 1.5em;
+$h3-font-size: 1.125em;
+$h4-font-size: 1em;
+
+$h1-font-size-lg: 1.5em;
+$h2-font-size-lg: 1.5em;
+$h3-font-size-lg: 1.125em;
+$h4-font-size-lg: 1em;
+
+//line height
+$h1-line-height: 2.625em;
+$h2-line-height: 2.375em;
+$h3-line-height: 1.75em;
+$h4-line-height: 1.25em;
+
+$h1-line-height-lg: 1.75em;
+$h2-line-height-lg: 1.75em;
+$h3-line-height-lg: 1.375em;
+$h4-line-height-lg: 1.25em;
```

Comment:
2 errors:

proportions must be taken from the element font size not the the body font size, because line-height em is always referred to the same element font-size value
you are mixing desktop values with mobile ones

Reply:
comments above resolved